src/main/resources/assets/css/main.less

```diff
@@ -570,6 +570,43 @@ fieldset[disabled] .btn-info.active {
   background-color: #8e929b;
 }
 
+.health-bar-column {
+  vertical-align: middle;
+
+  .health-bar {
```

Comment:
Will leave the color as literals. The main.less is full of those and I agree there should be variables for consistency. But not in this pr....